app/controllers/projects_controller.rb

```diff
@@ -76,7 +76,7 @@ def about
   end
 
   def dependents
-    @dependents = @project.dependent_projects.visible.paginate(page: page_number)
+    @dependents = @project.dependent_projects_v2.visible.paginate(page: page_number)
```

Comment:
My plan was to just deploy this, and if it helps, replace the other spots with this new method.